app/controllers/work_packages/moves_controller.rb

```diff
@@ -33,46 +33,24 @@ class WorkPackages::MovesController < ApplicationController
   before_filter :authorize
 
   def new
-    prepare_for_work_package_move
+    WorkPackageBulkUpdateService.new(@work_packages, @projects)
+      .run(params).available_attributes.each do |key, value|
+        instance_variable_set(:"@#{key}", value)
+      end
```

Comment:
I don't see why calling `instance_variable_set` in a loop is a better idea than explicitly setting three instance variables by hand.